pkg/labels/matcher.go

```diff
@@ -105,7 +105,13 @@ func (ms Matchers) Less(i, j int) bool {
 	if ms[i].Value > ms[j].Value {
 		return false
 	}
-	return ms[i].Value < ms[j].Value
+	if ms[i].Value < ms[j].Value {
+		return true
+	}
+	if ms[i].Type > ms[j].Type {
+		return false
+	}
+	return ms[i].Type < ms[j].Type
```

Comment:
Similar to what @vladimiroff said before, it's enough to just `return ms[i].Type < ms[j].Type`. The other three lines are redundant.